packages/terra-list/src/terra-dev-site/doc/guides/SectionList.jsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
 import List, {
   Item,
   Section,
   Utils,
-} from 'terra-list/lib/index'; // eslint-disable-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+} from 'terra-list/lib/index';
+/* eslint-enable */
```

Comment:
These should probably enable the rule that was disabled 

```
/* eslint-enable import/no-extraneous-dependencies */
```

Reply:
This was changed in e9a5d37a7c9efac19c5fa440f1777cc6d0f54f75